cmd/config/repo.go

```diff
@@ -0,0 +1,32 @@
+package config
+
+import (
+	"context"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Shopify/voucher/repository"
+	"github.com/Shopify/voucher/repository/github"
+)
+
+func newRepositoryClient(org repository.Organization) (repositoryClient repository.Client) {
+	if strings.Contains(org.URL, "github.com") {
```

Comment:
```suggestion
	 orgURL, err := url.Parse(org.URL)
	 if nil != err {
	 	 log.Errorf("failed to get github token: %s", err)
 		 return nil
 	}
 	
	if strings.HasSuffix(orgURL.Hostname, "github.com") {
```

Inline with my suggestion below, we should probably parse the URL so we don't end up treating `catherinesmaliciousserver.com/github.com/Shopify/voucher` as a GitHub URL!

Reply:
Done.